packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/DateTimePickerWithTimezoneMissingHour.test.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import DateTimePickerTimezoneExampleTemplate from '../../common/DateTimePickerTimezoneExampleTemplate';
+
+const DateTimePickerExample = () => (
+  <DateTimePickerTimezoneExampleTemplate
+    initialTimeZone="America/New_York"
```

Comment:
nit: The name of this test file shouldn't be `DateTimePickerWithTimezoneMissingHour` since it has nothing with missing hour. It is a test that can be used to test the missing hour scenario. Are we able to just use one of the other existing tests (e.g.; DateTimePickerWithTimezoneDst) to test this scenario so we don't have to create an extra one? The test steps might just need to clear the default value and then enter the value with the missing hour.

Reply:
changed here 15688a34fa0eff270ee5c526ff6e65624b2d5fa1